docs/07-developers.md

```diff
@@ -2,57 +2,75 @@
 
 Requirements:
 
-* Node.js current (8.x)
-* yarn >= 1.0.2
+- Node.js current (8.x)
+- yarn >= 1.0.2 || current stable (1.10.1)
 
-We recommend you to use [Node Version Manager](https://github.com/creationix/nvm) and to run `nvm install` followed by `nvm use` to get the right Node.js version.
+We recommend you to use [Node Version Manager](https://github.com/creationix/nvm) and to run `nvm install` in the root followed by `nvm use` to get the right Node.js version.
```

Comment:
Today, we can also upgrade Node to the new LTS, v10 🎉 https://github.com/nodejs/Release#release-schedule 
(not mandatory but good to have)